### PR TITLE
Add MemberPage class

### PR DESCRIPTION
**Pull request by chrismytton**
This adds a `MemberPage` class representing a single member's page. I've started with this class because it doesn't have any dependencies, so can be tested in isolation.

Note that this has currently _only_ been tested with the two people in the test fixtures.

This was largely taken from the previous attempt to port the scraper to Scraped, but I've removed two fields that didn't make sense in this new scraper:

- iddisputado - this wasn't actually being used in EveryPolitician
- phone/fax - neither of these were picking anything up in the previous scraper

Part of https://github.com/everypolitician/everypolitician-data/issues/16427

### CODE REVIEW

File: lib/member_page.rb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  end
+
+  field :constituency do
+    seat[/Diputad. por (.*)\./, 1]
+  end
+
+  field :photo do
+    foto = noko.at_css('#datos_diputado img[name="foto"]')
+    return if foto.nil?
+    foto[:src]
```
Comment (tmtmtmtm) [inline lib/member_page.rb]: Is there a reason this isn't just:

```ruby
  field :photo do
    noko.at_css('#datos_diputado img[name="foto"]/@src').text
  end
```

File: lib/member_page.rb [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  def query_string
+    URI.decode_www_form(URI.parse(url).query).to_h
+  end
+
+  def seat
+    noko.at_css('div#curriculum div.texto_dip ul li div.dip_rojo:first').text.tidy
+  end
+
+  def group
+    noko.at_css('div#curriculum div.texto_dip ul li div.dip_rojo:last').text.tidy
```
Comment (tmtmtmtm) [inline lib/member_page.rb]: there's a lot of duplication here that would probably be worth pulling out to a separate `scope`, even is that's only going to be a normal private method for now.

File: lib/member_page.rb (hunk 1 above)
Comment (chrismytton) [reply lib/member_page.rb]: Erm, no, no reason! Have switched to your nice concise version in https://github.com/everypolitician-scrapers/spain-congreso/pull/1/commits/fd0f82644ab29da1dc8a57daeabdcee130226171.

File: lib/member_page.rb (hunk 2 above)
Comment (chrismytton) [reply lib/member_page.rb]: Yep good shout. I've pulled that out into a private method in https://github.com/everypolitician-scrapers/spain-congreso/pull/1/commits/705a19e82d3ddae2108c96002d99da08a972d656.

**Review by tmtmtmtm — APPROVED** [review]
I suspect there are a few bits of this that are going to need to change as we add more classes, and check that this works against more pages, but 👍  for this as a launch pad.

I'll leave it to you to squash the fixup and merge.

**Pull request merged**